apps/docs/spec/Makefile

```
@@ -81,14 +88,31 @@ dereference.analytics.v0:
 generate.sections.api.v1:
 	npx tsx $(REPO_DIR)/sections/generateMgmtApiSections.cts \
 		$(REPO_DIR)/transforms/api_v1_openapi_deparsed.json \
 		$(REPO_DIR)/transforms/api_v2_openapi_deparsed.json \
 		$(REPO_DIR)/common-api-sections.json
 
+# Renders the "Personal Access Tokens" guide's permission tables (permission -> Management API
+# endpoints + MCP tools) from the OpenAPI specs, the MCP tool permission map, and the shared
+# permission catalog (packages/shared-data/scoped-access-token-permissions.ts, also the data
+# behind Studio's scoped personal access token form). Generating from the enforcement sources
+# means the tables can't drift from what mgmt-api enforces or what Studio calls each permission.
+generate.partials.access-control:
```

checkmake (0.3.2)

[warning] Target body for "generate.partials.access-control" exceeds allowed length of 5 lines (10).

(maxbodylength)